libass/ass_utils.c

```diff
@@ -56,8 +56,65 @@ char *ass_strndup_fallback(const char *s, size_t n)
 }
 #endif
 
+#if (defined(_WIN32) || defined(__APPLE__) || defined(__linux__) || defined(__FreeBSD__)) && \
+    (defined(__i386__) || defined(__x86_64__) || defined(__arm__) || defined(__aarch64__)) && \
+    (defined(__GNUC__))
+#define ALIGNED_OVERREADS_SAFE 1
+#endif
+
+#if __has_attribute(vector_size) && defined(ALIGNED_OVERREADS_SAFE)
```

Comment:
Optional suggestion: you may as well just use `ALIGNED_OVERREADS_SAFE` instead of `defined(ALIGNED_OVERREADS_SAFE)` here and elsewhere. Same effect (all undefined names are replaced by zeros), reads arguably better, and potentially allows the macro to be explicitly defined to 0, should anyone in the future want to do so.

Alternatively, if you do use `defined(...)`, you may as well `#define` it to an empty value rather than `1`.